modules/virt-automatic-certificates-frequency.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-automatic-certificates-frequency.adoc
+//
+
+[id="virt-automatic-certificates-frequency_{context}"]
+= Frequency of TLS certificate renewal
+
+|===
+|Component | Renewal frequency
+
+|KubeVirt
+|Daily.
+
+|Containerized data importer controller (CDI)
+|Every 15 days.
+
+
+|MAC pool
+|After one year.
```

Comment:
@phoracek this one might be too long to say that we entirely don't mention manual cert rotation. What do you think, should we document manual rotation for MAC pool, do you think it is ok, or should the rotation interval be lowered?

Reply:
We are working in CA controller improvements to make this configurable and lower the default interval as a part of that (CNV 2.5).

@qinqon could you please provide a script (as short as possible) to force the rotation?

Reply:
@phoracek  Is not enough to wait for CNV 2.5 ? or we just put manual script here for CNV 2.4 ?

Reply:
Just a manual script in case somebody wants to force the rotation. If we can have one.

For 2.5 we would revisit this, and replace by the proper thing.

Reply:
We have one already at kube-admission-webhook lib https://github.com/qinqon/kube-admission-webhook/blob/master/hack/force-cert-retation.sh

Reply:
Since cert rotation works entirely without the need for site admins to pay attention to it or take any action at all, why would we want to focus on manual rotation in the documentation? I feel that providing a script isn't necessary and increases risk of inadvertent issues.